shell-completion/bash/systemd-confext

```diff
@@ -66,7 +66,7 @@ _systemd-confext() {
                 comps=''
                 ;;
             --mutable)
-                comps='no yes auto import ephemeral ephemeral-import'
+                comps='no yes auto import ephemeral ephemeral-import help'
```

Comment:
Then, you can use the command and not necessary to list the candidates.
See e.g. completion for `journalctl --output=` in `shell-completion/bash/journalctl`.